website/docs/r/victor_ops_integration.html.markdown

```diff
@@ -1,34 +1,34 @@
 ---
 layout: "signalfx"
-page_title: "SignalFx: signalfx_victor_ops_integration"
+page_title: "Splunk Observability Cloud: signalfx_victor_ops_integration"
 sidebar_current: "docs-signalfx-resource-victor-ops-integration"
 description: |-
-  Allows Terraform to create and manage SignalFx VictorOps Integrations
+  Allows Terraform to create and manage Splunk On-Call Integrations
 ---
 
 # Resource: signalfx_victor_ops_integration
 
-SignalFx VictorOps integration.
+Splunk On-Call integrations.
```

Comment:
Is the name VictorOps completely retired now? It is kind of confusing how the resource is still called VictorOps but the name is "Splunk On-Call". I would maybe at least reference the old name in parenthesis for all of the devops people out there who don't keep up with the latest marketing updates.

Reply:
AFAIK, yes. Is that right, @mfoulds-splunk?